### PR TITLE
Adding support for the issue created: #2279. 

### PR DESCRIPTION
**Pull request by jayRyan24**
Adds in a collecting room to athletics. 

#2279.

### CODE REVIEW

**Comment by jayRyan24** [comment]
Working well for me so far.

**Comment by jayRyan24** [comment]
@rpherbig Seems pretty straight forward. Am using it currently.

File: profiles/base.yaml [hunk 1]
```diff
@@ -251,6 +251,7 @@ appraisal_training:
 full_pouch_container:
 classes_to_teach:
 avoid_athletics_in_justice: false
+athletics_outdoorsmanship_rooms:
```
Comment (rpherbig) [inline profiles/base.yaml]: If this is an array, it needs an entry in `base-empty.yaml`

Comment (jayRyan24) [reply profiles/base.yaml]: Done. What's this actually do?

Comment (rpherbig) [reply profiles/base.yaml]: If someone overrides a setting in their YAML such as `athletics_outdoorsmanship_rooms:` then `get_settings.athletics_outdoorsmanship_rooms` will return `nil`, not an array. Sometimes that's ok, but sometimes that causes an error (e.g. if you're using an Array-specific function in your script).

Base-empty is almost solely for settings which are arrays and hashes. It provides a default value if the setting is `nil` when loaded.

Comment (jayRyan24) [reply profiles/base.yaml]: Ah, thanks for the explanation.

**Pull request merged**